### ISSUE TITLE
DevTools Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by mfrascati** [GoogleChrome/lighthouse#6004]
**Initial URL**: https://unaparolaalgiorno.it/
**Chrome Version**: 69.0.3497.81
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
```
TypeError: Failed to construct 'URL': Invalid URL
    at new URLShim (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:4878:1)
    at NetworkRequest.onRequestWillBeSent (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1405:67)
    at NetworkRecorder.onRequestWillBeSent (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1384:151)
    at NetworkRecorder.dispatch (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1394:66)
    at devtoolsLog.forEach.message (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1397:126)
    at Array.forEach (<anonymous>)
    at Function.recordsFromLogs (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1397:93)
    at Function.afterPass (chrome-devtools://devtools/remote/serve_file/@032b3ca19e9af20182f9bd03deefc0faf4695558/audits2_worker/audits2_worker_module.js:1087:163)
```

**Comment by exterkamps** [comment]
What method did you use to get this?  DevTools Audit panel directly, extension, or the cli?

I was able to run an audit for [https://unaparolaalgiorno.it/](https://unaparolaalgiorno.it/) using cli and with devTools, but I am on a slightly newer Chrome Version, do you have steps to replicate?

Sometimes this can be a result of extensions in your browser itself, can you try to run this in incognito as well and see if the problem persists?

**Comment by patrickhulce** [comment]
closing as a now fixed duplicate of #6145 :)
